### PR TITLE
Kgpe test description

### PR DESCRIPTION
**Pull request by kl3mdeb**

### CODE REVIEW

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-boot.md [hunk 1]
```diff
@@ -0,0 +1,104 @@
+# Dasharo Compatibility: USB Boot
+
+## Test cases
+
+### UBB001.001 USB detect and boot after coldboot
+
+**Test description**
+
+This test aims to verify that the DUT properly detects USB device and boots into
+operating system after coldboot.
```
Comment (sulewskiprzemyslaw) [inline docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-boot.md:10]: In the test case desription we should add information, that the test case may be re-done several times to specify the platform and connection stability.

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-boot.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+**Expected result**
+
+The `Debian GNU/Linux` is visible and confirms successful boot into OS after
+coldboot.
+
+### UBB001.002 USB detect and boot after warmboot
+
+**Test description**
+
+This test aims to verify that the DUT properly detects USB device and boots into
```
Comment (sulewskiprzemyslaw) [inline docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-boot.md:41]: In the test case desription we should add information, that the test case may be re-done several times to specify the platform and connection stability.

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-boot.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+**Expected result**
+
+The `Debian GNU/Linux` is visible and confirms successful boot into OS after
+warmboot.
+
+### UBB001.003 USB detect and boot after system reboot
+
+**Test description**
+
+This test aims to verify that the DUT properly detects USB device and boots into
```
Comment (sulewskiprzemyslaw) [inline docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-boot.md:72]: In the test case description we should add information, that the test case may be re-done several times to specify the platform and connection stability.

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

**Comment by kl3mdeb** [comment]
Added improvements.

File: docs/variants/asus_kgpe_d16/dasharo-security/tpm-support.md [hunk 4]
```diff
@@ -0,0 +1,134 @@
...
+    
+    ```
+    apt install tpm2-tools
+    ```
+11. For TPM2.0 - Execute command.
+
+```
+tpm2_pcrread
+```
+1.  For TPM1.2 - Execute command:
```
Comment (sulewskiprzemyslaw) [inline docs/variants/asus_kgpe_d16/dasharo-security/tpm-support.md:43]: @kl3mdeb take a look at the numeration, please

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-boot.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
+# Dasharo Compatibility: USB Boot
+
+## Test cases
+
+### UBB001.001 USB detect and boot after coldboot
+
+**Test description**
+
+This test aims to verify that the DUT properly detects USB device and boots into
```
Comment (sulewskiprzemyslaw) [inline docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-boot.md:9]: We should rewrite description like in USB test cases

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-boot.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+
+The `Debian GNU/Linux` is visible and confirms successful boot into OS after
+coldboot.
+
+### UBB001.002 USB detect and boot after warmboot
+
+**Test description**
+
+This test aims to verify that the DUT properly detects USB device and boots into
+the operating system after warmboot. In this test case powering on/off is done
```
Comment (sulewskiprzemyslaw) [inline docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-boot.md:44]: We should rewrite description like in USB test cases

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-boot.md [hunk 7]
```diff
@@ -0,0 +1,114 @@
...
+The `Debian GNU/Linux` is visible and confirms successful boot into OS after
+warmboot.
+
+### UBB001.003 USB detect and boot after system reboot
+
+**Test description**
+
+This test aims to verify that the DUT properly detects USB device and boots into
+the operating system after system reboot. In this test case powering on/off is
+done by RTE relay. This test case may be re-done several times to specify the
```
Comment (sulewskiprzemyslaw) [inline docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-boot.md:78]: We should rewrite description like in USB test cases

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-detect.md [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+2. `BOOT_MENU_STRING` = `Press ESC for boot menu`
+3. `USB_STICK` = `USB MSC Drive General USB Flash Disk 1100`
+
+**Test setup**
+
+1. Proceed with the [Generic test setup: firmware](generic-test-setup.md#firmware).
+
+**Test steps**
+
+1. Power off the DUT.
```
Comment (sulewskiprzemyslaw) [inline docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-detect.md:26]: We should replace in the Test steps section the following statements: Power On/Off to Power cutting On/Off.

File: docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-boot.md [hunk 9]
```diff
@@ -0,0 +1,114 @@
...
+2. `BOOT_MENU_STRING` = `Press ESC for boot menu`
+3. `USB_STICK` = `USB MSC Drive General USB Flash Disk 1100`
+
+**Test setup**
+
+1. Proceed with the [Generic test setup: firmware](generic-test-setup.md#firmware).
+
+**Test steps**
+
+1. Power off the DUT.
```
Comment (sulewskiprzemyslaw) [inline docs/variants/asus_kgpe_d16/dasharo-compatibility/usb-boot.md:26]: We should replace in the Test steps section the following statements: Power On/Off to Power cutting On/Off.

**Review by sulewskiprzemyslaw — CHANGES_REQUESTED** [review]

**Comment by kl3mdeb** [comment]
Just added improvements to usb-boot and usb-detect descriptions

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Review by sulewskiprzemyslaw — APPROVED** [review]

**Pull request merged**